das/state.go

```diff
@@ -77,14 +103,19 @@ func (s *coordinatorState) handleResult(res result) {
 
 	// update failed heights
 	for h := range res.failed {
-		failCount := 1
-		if res.job.jobType == retryJob {
-			// if job was already in retry and failed again, persist attempt count
-			failCount += s.inRetry[h]
+		// if job was already in retry and failed again, persist attempt count
+		lastRetry, ok := s.inRetry[h]
+		if ok {
 			delete(s.inRetry, h)
 		}
 
-		s.failed[h] = failCount
+		nextRetry, retryExceeded := s.retryStrategy.nextRetry(lastRetry, time.Now())
+		if retryExceeded {
+			log.Warnw("header exceeded maximum amount of sampling attempts",
```

Comment:
so basically all that happens here is that we log a warning if max retries were exceeded but we still keep retrying the height at the last backoff interval rate forever? 

IMO we should have some sort of special categorisation inside daser state that will indicate if height is truly unavailable that would be very apparent when querying das SamplingStats but that can be left for another PR.

Reply:
Thats a good idea and it could be done on representation layer of daser stats. I'll create and issue for it